### PR TITLE
added simple transformation script

### PR DESCRIPTION
**Pull request by aayush-k**

### CODE REVIEW

File: transformation/pairUp.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+avoid = {'vaish': ['aayush', 'kaela'], 'aayush': [], 'nathan': ['kaela', 'vaish']}
+assign = {'vaish': None, 'aayush': None, 'kaela': None, 'Nathan': None}
+available = ['vaish', 'aayush', 'nathan', 'kaela']
+
+def match(remaining, person, avoid):
```
Comment (SujeethJinesh) [inline transformation/pairUp.py]: Lol, you left something in here :P

File: transformation/transform.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import cv2
+import numpy as np
+
+def transform(img):
```
Comment (SujeethJinesh) [inline transformation/transform.py:4]: Could you add some comments? Makes it a little clearer

File: transformation/pairUp.py (hunk 1 above)
Comment (aayush-k) [reply transformation/pairUp.py]: whoops

File: transformation/transform.py (hunk 2 above)
Comment (aayush-k) [reply transformation/transform.py:4]: 👍

**Comment by SujeethJinesh** [comment]
👍

**Review by SujeethJinesh — APPROVED** [review]
👍

**Pull request merged**